### PR TITLE
fix typo in how to set custom elfeed.org

### PR DESCRIPTION
**Pull request by jrmiller82**
Per https://github.com/remyhonig/elfeed-org, the `modules/app/RSS` README.org had the incorrect command for setting a custom elfeed.org file. It was missing a `list` in the elisp.

### CODE REVIEW

File: modules/app/rss/README.org [hunk 1]
```diff
@@ -68,7 +68,7 @@ When using ~+org~ flag then configuration is easier. You can use ~org-mode~ to c
 + Root of ~elfeed-org~ needs to have ~:elfeed:~ tag. This is where ~elfeed-org~ starts to read.
 + You can have subheaders as in example ~:programming:~, and ~elfeed-org~ applies that tag to all subheader feeds, in example it adds it to ~This Week in Rust~.
 + You can "name" feeds as you please with ~org-mode~ ~org-insert-link~ (~C-c C-l~) and put name as you want into ~description~.
-+ If you don't want to use ~org-directory/elfeed.org~ file you can specify it with ~(setq rmh-elfeed-org-files ("path/to/your/elfeed/file.org))~
++ If you don't want to use ~org-directory/elfeed.org~ file you can specify it with ~(setq rmh-elfeed-org-files (list "path/to/your/elfeed/file.org))~
```
Comment (hlissner) [inline modules/app/rss/README.org:71]: Please quote it instead: 

```diff
- If you don't want to use ~org-directory/elfeed.org~ file you can specify it with ~(setq rmh-elfeed-org-files (list "path/to/your/elfeed/file.org))~
+ If you don't want to use ~org-directory/elfeed.org~ file you can specify it with ~(setq rmh-elfeed-org-files '("path/to/your/elfeed/file.org))~
```

**Review by hlissner — CHANGES_REQUESTED** [review]

File: modules/app/rss/README.org (hunk 1 above)
Comment (jrmiller82) [reply modules/app/rss/README.org:71]: I was just using the exact example from upstream; I'll update shortly.

Comment (jrmiller82) [reply modules/app/rss/README.org]: Fixed.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks!

**Comment by jrmiller82** [comment]
You're welcome! Thanks for the work you do on doom emacs!

**Pull request merged**